src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -303,22 +349,6 @@ function ExpandAggregateRow(props: {
     });
   }
 
-  // count(column) drilldown
-  if (aggregation === 'count') {
```

Comment:
Wouldn't we want to keep this?

Reply:
@markstory  I think it was confirmed by Dora and Bowen that we remove this in light of the new navigation flow. 

In a follow up PR, I'll be adding the per-cell actions for `count(column)`; which was disabled temporarily until the navigation flow was implemented.


/cc  @doralchan @bowencai8